mcp_server_gravitino/server/tools/common_tools.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Datastrato Pvt Ltd.
+# This software is licensed under the Apache License version 2.
+from typing import List, Optional, Tuple
+
+
+def parse_four_level_fqn(fqn: List[str]) -> Tuple[Optional[str], str, str, str]:
+    """
+    Parse a fully qualified model name into its components.
```

Comment:
please use "entity name" to replace the "model name", as this method is a common method now.